update.sh

```diff
@@ -1,15 +1,30 @@
 #!/bin/bash
 
-cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
+# Take two versions as strings (v1, v2), return 1 if v1 > v2 numerically, otherwise return 0.
```

Comment:
I think this comment is wrong.

Reply:
I have removed this comment as the new method name is self explanatory.